content/en/examples/secret/ssh-auth-secret.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-ssh-auth
+type: kubernetes.io/ssh-auth
+data:
+  # the data is abbreviated in this example
+  ssh-privatekey: |
+    UG91cmluZzYlRW1vdGljb24lU2N1YmE=
```

(nit)
```suggestion
    LS0tLS1CRUdJTiBFTkNSWVBURUQgUFJJVkFURSBLRVktLS0tLQo=
```

&hellip;try base64 decoding it - you'll get:
```
-----BEGIN ENCRYPTED PRIVATE KEY-----
```